internal/publish/model/exposure_model.go

```diff
@@ -249,7 +249,8 @@ func (t *Transformer) TransformPublish(ctx context.Context, inData *verifyapi.Pu
 	nextInterval := entities[0].IntervalNumber
 	for _, ex := range entities {
 		if ex.IntervalNumber < nextInterval {
-			return nil, fmt.Errorf("exposure keys have overlapping intervals")
+			logging.FromContext(ctx).Errorf("Server detected keys with overlapping intervals")
```

Comment:
We might base it on whether debugReleaseSameDay is true, since this should only arise in testing.

Reply:
I pushed an update to gate this on the debug config.